### PR TITLE
pokecli.py: wait_time for every exception randomized

### PR DESCRIPTION
**Pull request by mudmud2k**
Feature Request (https://github.com/PokemonGoF/PokemonGo-Bot/issues/5579)

Short Description:
it's create a random waiting time based on config for all exception (connection lost, throttling, and so on), so no static time span will be used anymore
---

randint(min,max)
config.reconnecting -> min
config.reconnection \* 60 -> max

### CODE REVIEW

**Comment by solderzzc** [comment]
👍

**Comment by YvesHenri** [comment]
Can't review now. Someone please do it and merge, if agreed.

File: pokecli.py [hunk 1]
```diff
@@ -185,7 +186,7 @@ def get_commit_hash():
         finished = False
 
         while not finished:
-            wait_time = config.reconnecting_timeout * 60
+            wait_time = randint(config.reconnecting_timeout, (config.reconnecting_timeout * 60))
```
Comment (DBa2016) [inline pokecli.py]: I think this needs adjustment and docs changes: previously, config.reconnecting_timeout was "time in minutes". Now the wait_time spans from this valud to the 60-times of it.

Maybe `randint(config.reconnecting_timeout*60, (config.reconnecting_timeout + 1) * 60))` ?

File: pokecli.py [hunk 2]
```diff
@@ -241,9 +242,10 @@ def get_commit_hash():
                     'api_error',
                     sender=bot,
                     level='info',
-                    formatted='Server is throttling, reconnecting in 30 seconds'
+                    formatted='Server is throttling, reconnecting in {:d} seconds'.format(wait_time)
                 )
-                time.sleep(30)
+                time.sleep(wait_time)
+#                 sys.exit()
```
Comment (DBa2016) [inline pokecli.py]: should be removed

**Comment by BreezeRo** [comment]
Any updates on this?

**Comment by mudmud2k** [comment]
still waiting for a review, or further discussion...

**Comment by solderzzc** [comment]
👍

**Comment by Gobberwart** [comment]
This doesn't work as expected... 

wait_time = randint((config.reconnecting_timeout * 0.8 * 60), (config.reconnecting_timeout *  1.2 * 60))

randint requires two INTEGER values, whereas this line provides two FLOATS. Result:

```
Traceback (most recent call last):
  File ".\pokecli.py", line 844, in <module>
    main()
  File ".\pokecli.py", line 189, in main
    wait_time = randint((config.reconnecting_timeout * 0.8 * 60), (config.reconnecting_timeout *  1.2 * 60))
  File "C:\Python27\lib\random.py", line 242, in randint
    return self.randrange(a, b+1)
  File "C:\Python27\lib\random.py", line 187, in randrange
    raise ValueError, "non-integer arg 1 for randrange()"
ValueError: non-integer arg 1 for randrange()
```

**Comment by Gobberwart** [comment]
Correction: works if reconnecting_timeout in config is set to default 15, or any multiple of 5, but fails if set to anything else. (5 x 0.8 = 12 (int), 5 x 1.2 = 18 (int). In my case, I have reconnecting_timeout set to 3, so 3 x 0.8 = 2.4 (float), 3 x 1.2 = 3.6 (float).

**Pull request merged**